src/snarky.d.ts

```diff
@@ -51,489 +53,806 @@ declare interface ProvablePure<T> extends Provable<T> {
 }
 
 /**
- * An element of a finite field.
+ * A {@link Field} is an element of a prime order [finite field](https://en.wikipedia.org/wiki/Finite_field).
+ * Every other provable type is built using the {@link Field} type.
+ *
+ * The field is the [pasta base field](https://electriccoin.co/blog/the-pasta-curves-for-halo-2-and-beyond/) of order 2^254 + 0x224698fc094cf91b992d30ed00000001 ({@link Field.ORDER}).
+ *
+ * You can create a new Field from everything "field-like" (`bigint`, integer, `number`, decimal, `string`, `Field`).
+ * @example
+ * ```
+ * Field(10n); // Field contruction from a big integer
+ * Field(100); // Field construction from a number
+ * Field("1"); // Field construction from a decimal string
+ * ```
+ *
+ * **Beware**: Fields can _not_ be constructed from fractional numbers or alphanumeric strings:
```

Comment:
```suggestion
 * **Beware**: Fields _cannot_ be constructed from fractional numbers or alphanumeric strings:
```

Reply:
for consistency, let's make cannot a single word 

https://www.merriam-webster.com/words-at-play/cannot-vs-can-not-is-there-a-difference